[2025-03-31T08:15:34.910+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T08:15:34.935+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-31T08:15:34.947+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-31T08:15:34.949+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T08:15:34.964+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-03-27 00:00:00+00:00
[2025-03-31T08:15:34.972+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1892) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-31T08:15:34.973+0000] {standard_task_runner.py:72} INFO - Started process 1896 to run task
[2025-03-31T08:15:34.972+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2025-03-27T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpcv1a_ipb']
[2025-03-31T08:15:34.974+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask second_task
[2025-03-31T08:15:35.023+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2025-03-27T00:00:00+00:00 [running]> on host 9663692256f6
[2025-03-31T08:15:35.108+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T00:00:00+00:00'
[2025-03-31T08:15:35.109+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T08:15:35.126+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T08:15:35.127+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will execute after task1!']
[2025-03-31T08:15:35.136+0000] {subprocess.py:99} INFO - Output:
[2025-03-31T08:15:35.138+0000] {subprocess.py:106} INFO - hey, I am task2 and will execute after task1!
[2025-03-31T08:15:35.138+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T08:15:35.164+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T08:15:35.165+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=scheduled__2025-03-27T00:00:00+00:00, execution_date=20250327T000000, start_date=20250331T081534, end_date=20250331T081535
[2025-03-31T08:15:35.199+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T08:15:35.212+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
